### PR TITLE
Add SwiftUI Search Entry

### LINKED ISSUE #2104: [Concept Entry] SwiftUI Search
### Reason/inspiration (optional)

We would like a new concept entry for SwiftUI Search.

The edit should go in an existing file in docs/content/swiftui/concepts/search/search.md 

The entry should:

- Explain how to create a Search Interface in SwiftUI, starting with an introductory pararaph.
- Include headers with sections on how to create a search interface by applying searchable view modifier.
- Not a requirement to go into too much detal about tokens or queries; a general overview is sufficient! See: [Apple Docs on Search](https://developer.apple.com/documentation/swiftui/adding-a-search-interface-to-your-app)

Please refer to the [entry template](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by gracekishino**
### Description

Added new folder `search` inside `content/swiftui/concepts/`.
Added new entry file `content/swiftui/concepts/search/search.md` for SwiftUI Search.
Added 2 new images for the SwiftUI Search entry `swiftui-search-1.jpg` and `swiftui-search-2.jpg`.

Closes #2104

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/search/search.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    .searchable(
+        text: binding variable of text to search for,
+        placement: position (optional),
+        prompt: prompt text (optional)
+    )
+```
+
+Here are a few of the `placement` property types:
+
+- `.automatic` places the search box will be automatically, this is the default value.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/search/search.md:30]:
```suggestion
- `.automatic` places the search box automatically, this is the default value.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@gracekishino I did a first review and only had two minor comments.

**Comment by gracekishino** [comment]
@SSwiniarski thank you for reviewing! I committed the changes.

**Review by SSwiniarski — APPROVED** [review]
@gracekishino looks good for a second review.

File: content/swiftui/concepts/search/search.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        prompt: prompt text (optional)
+    )
+```
+
+Here are a few of the `placement` property types:
+
+- `.automatic` places the search box automatically, this is the default value.
+- `.sidebar` places the search box in the sidebar.
+- `.toolbar` places the search box in the toolbar.
+
```
Comment (KTom101) [inline content/swiftui/concepts/search/search.md:33]: Can we have an explanation of what `prompt` is?

**Review by KTom101 — CHANGES_REQUESTED** [review]
@gracekishino I've added some review comments!

File: content/swiftui/concepts/search/search.md (hunk 2 above)
Comment (gracekishino) [reply content/swiftui/concepts/search/search.md:33]: I added in an explanation of `prompt`.

**Comment by gracekishino** [comment]
@KTom101 thanks so much for reviewing! I added in the suggested changes.

**Review by KTom101 — APPROVED** [review]
@gracekishino LGTM 👍

**Comment by KTom101** [comment]
@gracekishino Entry is live: https://www.codecademy.com/resources/docs/swiftui/search 🎉

Screenshot 2023-07-02 at 5 56 17 PM

cc @yangc95

**Comment by gracekishino** [comment]
@KTom101 @SSwiniarski thanks so much for your help with this entry!

**Comment by KTom101** [comment]
@gracekishino You're very welcome!

**Pull request merged**